TickSpec.XunitWiring/TickSpec.XunitWiring.fsproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+<!--  <Import Project="../netfx.props" />-->
```

Comment:
dont forget to tidy (or does it already work?)

Reply:
sorry, uncommented, but this is really error-prone, I tell you .. wish tickspec would be .net 5 only ...

Reply:
well TickSpec.Xunit2 can def be net50 only IMO

Reply:
Can this one be closed - I keep the netfx.props import just to be consistent with the other projects ...

Reply:
Up to you ;)